src/Libraries/RevitServices/Persistence/DocumentManager.cs

```diff
@@ -106,6 +123,21 @@ public IEnumerable<Element> ElementsOfCategory(BuiltInCategory category)
         }
 
         /// <summary>
+        /// This property represents the hash code for the current active Revit
+        /// Document object. If there is no active document, this value will be 
+        /// set to zero. This property is made an 'int' because it is primarily
+        /// meant to indicate the active document for comparison purposes, no 
+        /// Document specific operations should be allowed on it. This property 
+        /// is updated when a document switch or a document closure happens. The
+        /// reason this property is needed because the following property cannot
```

Comment:
@Benglin Why is this the case? Is the Revit API broken? If so, have we reported this to the Revit team?